README.md

```diff
@@ -1 +1,76 @@
-# Math-magicians-setup
\ No newline at end of file
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+## Built With
+
+- JavaScript
+- React
+
+## Getting Started
+
+1. Open [dev branch](https://github.com/Yemisirach)
+2. Open VSCode or terminal and navigate to the folder where you would like to clone the project.
+3. Type "pwd" to see your working directory.
+4. [clone repository](git@github.com:Yemisirach/Math-magicians-setup.git)
+
+```bash
+$ git clone git@github.com:Yemisirach/Math-magicians-setup.gitd
+```
+
+5. Run
+
+```bash
+  $ npm install # to install required packages
+  $ npm run start # to start the website on a local server
+```
+
+6. Enjoy 😎
+
+## Author 👱‍♂️
+
+👤 **Yemisirach Tamirat**
+
+- GitHub: [@yemisrach](https://github.com/Yemisirach)
+- Twitter: [@Yemisirach](https://twitter.com/tamiratyemsrach)
+- LinkedIn: [@Yemisirach](https://www.linkedin.com/in/yemisirach)
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[CSS styling]**
+- [ ] **[React Multi Pages]**
+- [ ] **[Server Storage]**
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Submit an issue
+
+[Issue page](https://github.com/yemisirach)
```

Comment:
- [ ] Please replace your issue link with the below link
https://github.com/Yemisirach/Math-magicians-setup/issues